docs/ossm/ossm2-migration/migration-checker.sh

```diff
@@ -26,8 +26,11 @@ BLANK='\033[0m'
 WARNING_EMOJI='\u2757'
 SPACER="-----------------------"
 
+# TODO: autodetect latest versions
 LATEST_VERSION=v2.6
 LATEST_CHART_VERSION=2.6.4
```

Comment:
```suggestion
LATEST_CHART_VERSION=2.6.6
```

Reply:
I'll take a stab at autodetecting these... I think trying to keep these up to date will be problematic.

Reply:
Added autodetection pulling from package manifests